tutorials/Native_distributions_and_local_execution/jpackage-conveyor-comparison.md

```diff
@@ -0,0 +1,25 @@
+# Packaging tools comparison 
+
+Use [Conveyor](https://www.hydraulic.software) if you want any of the following features:
```

Comment:
Maybe it's just me, but the comparison seems to favour Conveyor over the default plugin.
While I do think:
* that listing alternative options is useful for current and future users;
* Conveyor does provide a lot more features, and likely will be for the foreseeable future; 
it seems reasonable to describe what is provided out-of-the-box first,
and then describing, what Conveyor or other third-party tools improve. 

What do you think about restructuring this section to something like:
```
Compose Multiplatform Gradle plugin covers basic packaging options:
...

Consider using [Conveyor](https://www.hydraulic.software) (a third party tool developed by Hydraulic Software), if you want any of the following features:
...  
```